src/olympia/editors/tests/test_models.py

```diff
@@ -580,18 +577,19 @@ def test_award_points_bonus(self):
     def test_award_moderation_points(self):
         ReviewerScore.award_moderation_points(self.user, self.addon, 1)
         score = ReviewerScore.objects.all()[0]
-        eq_(score.score, amo.REVIEWED_SCORES.get(amo.REVIEWED_ADDON_REVIEW))
-        eq_(score.note_key, amo.REVIEWED_ADDON_REVIEW)
+        assert score.score == amo.REVIEWED_SCORES.get(
+            amo.REVIEWED_ADDON_REVIEW)
```

Comment:
Maybe something like that but I'm torn between all the possibilities so, maybe just ignore that :)